pyatlan/model/core.py

```diff
@@ -60,10 +54,13 @@ def to_snake_case(value):
 
 
 class AtlanTagName:
-    def __init__(self, display_text: str):
-        from pyatlan.cache.atlan_tag_cache import AtlanTagCache
-
-        if not AtlanTagCache.get_id_for_name(display_text):
+    def __init__(
+        self,
+        display_text: str,
```

Comment:
Add ability to pass client otherwise you need to remember to set the default_client correctly.